Library/Formula/tippecanoe.rb

```diff
@@ -23,8 +23,10 @@ def install
     path.write <<-EOS.undent
       {"type":"Feature","properties":{},"geometry":{"type":"Point","coordinates":[0,0]}}
     EOS
-    output = `#{bin}/tippecanoe -o test.mbtiles #{path}`.strip
+
+    `#{bin}/tippecanoe -o test.mbtiles #{path}`
```

Comment:
In that case just use `system` instead which checks the exit code so you can remove the line below.